src/main/java/reposense/frontend/CliArguments.java

```diff
@@ -121,6 +134,21 @@ private void setUserInputValuesToArgument(final HashMap<String, String> argument
         }
     }
 
+    /**
+     * Verifies the correctness of input values
+     *
+     * @throws IllegalArgumentException If user-supplied values are invalid
+     */
+    private void verifyInputValues() {
+
+        // Checks sinceDate and untilDate if they are provided
+        if (sinceDate.isPresent() && untilDate.isPresent()) {
+            if (sinceDate.get().getTime() > untilDate.get().getTime()) {
```

Comment:
We can just do this check in one line right?
No need for the comment too.

Reply:
I wanted to increase readability. As I believe the body of the method could lengthen in the future.